File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        hash_items = set(nums)
        count = 0

        for num in nums:
            i=0
            if num-1 in  hash_items:
                continue
            while((num+i) in hash_items):
                i += 1   
            count = max(count,i)
            
        return count

File: Data Structures & Algorithms/concatenation-of-array/submission-0.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int[] ans= new int[2*nums.Length];
        for(int i = 0; i <= ans.Length-1; i++)
        {
           
            ans[i] = nums[i % nums.Length];
           
        }
        return ans;
    }
}